src/components/report-viewer-tab.js

```diff
@@ -17,11 +17,20 @@ import FormControlLabel from '@mui/material/FormControlLabel';
 import Switch from '@mui/material/Switch';
 import { useIntl } from 'react-intl';
 import makeStyles from '@mui/styles/makeStyles';
+import Alert from '@mui/material/Alert';
+import { allowModificationsOnNode } from './graph/util/model-functions';
 
 const useStyles = makeStyles((theme) => ({
+    div: {
+        display: 'flex',
+    },
     reportOnlyNode: {
         margin: 5,
     },
+    invalidNode: {
+        marginTop: '4px',
+        marginLeft: '50px',
```

Comment:
Use theme.spacing(X) instead of pixels ?
theme.spacing(0.5) = 4px;
theme.spacing(6.25) = 50px;

Reply:
Done